packages/app/src/api.ts

```diff
@@ -272,9 +277,9 @@ export function initializeServerApp(
   const nameString = '' + hashCode(JSON.stringify(nameObj));
   const existingApp = _serverApps.get(nameString) as FirebaseServerApp;
   if (existingApp) {
-    // TODO:
-    //   1: Register a new reference to finalization registry.
-    //   2: Incrememnt reference count.
+    (existingApp as FirebaseServerAppImpl).incRefCount(
```

Comment:
The removed TODO comment of `Register a new reference to finalization registry.` is handled above.